bin/qemu.maltael

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2016 Konsulko Group. All rights reserved.
```

Comment:
I think this file can be dropped. Either that, of conf.malta64_qemu needs to set qemu_helper_script to point at this file?

Reply:
Oh, and I think you want to add-to/update all these copyright headers?

Reply:
Oh I see that some of the new conf files do use this but not all, based on endianness.

A good cleanup, perhaps later, would be to share code/logic between bin/qemu.* since they're the same except for input filename. Also, qemu.* isn't a good name; qemu-flash-gen.* might be better. But this can come later.

Reply:
Now I switched all qemu_mips and malta targets to script flash.create_image_32mb and removed the redundant qemu helper script. For flash.create_image_32mb I introduced a new parameter flash_u_boot_bin which the maltael and malta64el boards set to u-boot-swap.bin.

Reply:
I needed to add another parameter flash_size because newer Qemu version limit the CFI backend for qemu_mips and Malta to 4 MiB.